code/datums/gamemode/factions/wilkersons/wilkerson_spells.dm

```diff
@@ -0,0 +1,164 @@
+				H.adjustBruteLoss(kickDam)
+				if(prob(kickDam*10))
+					H.Knockdown(kickDam/2)
+
+
+
+//MALCOLM/////
+
+/spell/malcolmTimeStop
+	name = "Speak with Audience"
+	desc = "Stop time in an area around yourself so you can speak with the audience. During this time you can move freely."
+	deepLore = "Malcolm began his awakening when he first turned to the camera, the world around him stopped, and he spoke to the audience."
+	charge_max = 1200
+
+/spell/malcolmTimeStop/cast(var/list/targets, var/mob/user)
+	var/mob/living/carbon/monkey/malcolm/M = user
+	M.toggleMalcolmMovement(10)
+
+/spell/malcolmUnfair
+	name = "Life is Unfair"
+	desc = "Immediately teleport yourself, Dewey, and Reese back to The Middle. This spell can only be cast a finite number of times."
+	deepLore = "It's not chance or luck that makes Malcolm the Middle of All Things. That would imply there was any other option. No, there is only Malcolm."
+	charge_type = Sp_CHARGES
+	charge_max = 3
+
+/spell/malcolmUnfair/cast(var/list/targets, var/mob/user)
+	var/mob/living/carbon/monkey/malcolm/M = user
+	M.forceMove(get_turf(M.theMiddle))	//Not even teleport blocking is possible. Life truly is unfair.
+	for(var/datum/role/wilkerson/reese/R in M.mind.faction.members)
+		var/turf/T = get_turf(get_step(M.theMiddle, EAST))
+		R.host.forceMove(T)	//See it's theming
+	for(var/datum/role/wilkerson/dewey/D in M.mind.faction.members)
+		var/turf/T = get_turf(get_step(M.theMiddle, WEST))
+		D.host.forceMove(T)	//Because he's in the middle, high art
+
+/spell/malcolmBossOfMeNow
+	name = "You're not the Boss of Me Now"
+	deepLore = "Malcolm's defiant nature was usually explained as due to his age. Malcolm being exactly as old as the universe makes this seem unlikely."
+	charge_max = 30
+
+/spell/malcolmBossOfMeNow/cast(var/list/targets, var/mob/user)
+
+
+//DEWEY//////
+
+/spell/deweyVomit
+	name = "Recall Birth"
+	desc = "Cause all humans in the area to vomit. You and your brothers are no longer human, of course."
+	deepLore = "Malcolm, being older than Dewey, was just old enough to have partial memories of his birth. Malcolm fills in the gaps and attaches the uncomfortable associations to Dewey."
+	charge_max = 600
+
+/spell/deweyVomit/cast(var/list/targets, var/mob/user)
+	for(var/mob/living/carbon/human/H in view(9, user))
+		H.vomit()
+		H.Jitter(30)
+		H.stuttering += 30
+
+/spell/deweyPolyp
+	name = "Split"
+	desc = "Create a mindless copy of yourself."
+	deepLore = "Dewey was always around. Malcolm had to rationize it, and so it was as he perceived."
+	charge_max = 50
+
+/spell/deweyPolyp/cast(var/list/targets, var/mob/user)
+	var/mob/living/carbon/monkey/dewey/D = user
+	D.visible_message("<span class='warning'>[D] grows dislodges a chunk of flesh from himself!</span>")
+	D.splitDewey()
+
+/spell/deweyEnrage
+	name = "Forego Medication"
+	desc = "All Dewey's will enter a curious fervor. They will latch on to their victims and jump them back to your side."
```

Comment:
It should be All Deweys, meaning all the Dewey individuals, not All Dewey's, meaning all of Dewey's property.